ui/components/app/edit-gas-display/edit-gas-display.component.js

```diff
@@ -133,14 +133,6 @@ export default function EditGasDisplay({
   return (
     <div className="edit-gas-display">
       <div className="edit-gas-display__content">
-        {warning && !isGasEstimatesLoading && (
```

Comment:
I opted to take off the `warning` prop because I didn't see it getting used. We can very easily just add a warning ourselves to this component.